spec/helpers/ect_helper_spec.rb

```diff
@@ -10,11 +10,11 @@
 
     context "when the ECT is leaving" do
       before do
-        ect_at_school_period.update!(finished_on: Time.zone.today + 1.day)
+        ect_at_school_period.update!(finished_on: Time.zone.today + 1.day, reported_leaving_by_school_id: school.id)
       end
 
       it "returns a yellow 'Leaving school' tag" do
-        expect(helper.ect_status(ect_at_school_period)).to have_css("strong.govuk-tag.govuk-tag--yellow", text: "Leaving school")
+        expect(helper.ect_status(ect_at_school_period, current_school: school)).to have_css("strong.govuk-tag.govuk-tag--yellow", text: "Leaving school")
       end
```

Comment:
You could add in specs to cover 1) if `current_school` isn't passed, it should not show leaving and possibly 2) if a different school is passed it must not show leaving. Probably the first is enough

Reply:
Good shout - done!